central/notifiers/utils/encryption.go

```diff
@@ -0,0 +1,20 @@
+package utils
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	encryptionKeyFile = "/run/secrets/stackrox.io/notifiers/enc-key"
```

Comment:
@johannes94 where are you actually going to mounting the key to?

Reply:
According to this [PR](https://github.com/stackrox/stackrox/pull/7889/files) by Vlad it'll be `/run/secrets/stackrox.io/central-encryption-key` the current data field is `encryptionKey` so the full path to the file would be `/run/secrets/stackrox.io/central-encryption-key/encryption-key`.

If we want a different default path we can still change it easily in Vlad's PR.

Reply:
This is fine. I can change the path in my PR.

Reply:
Also @johannes94 , is the key stored as a plain string or as a base64 std/url encoded string?

Reply:
Resolved on slack.